config/other.go

```diff
@@ -169,11 +169,17 @@ func setDefaults(conf *Config, bus awlevent.Bus) {
 		conf.VPNConfig.InterfaceName = defaultInterfaceName
 	}
 
+	uniqAliases := make(map[string]struct{}, len(conf.KnownPeers))
 	if conf.KnownPeers == nil {
 		conf.KnownPeers = make(map[string]KnownPeer)
 	}
 	for peerID := range conf.KnownPeers {
 		peer := conf.KnownPeers[peerID]
+		newAlias := conf.genUniqPeerAlias(peer.Name, peer.Alias, uniqAliases)
+		if newAlias != peer.Alias {
+			logger.Warnf("incorrect config: peer alias %s is not unique, updated automaticaly to %s", peer.Alias, newAlias)
```

Comment:
I think we should also print the peerID here to simplify debugging, to know which of the same peers was renamed to which name